Corona-Warn-App/src/main/res/layout/tracing_content_increased_view.xml

```diff
@@ -120,7 +79,7 @@
             app:compatIconTint="@color/colorStableLight"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/row_tracing_days"
+            app:layout_constraintTop_toBottomOf="@+id/row_contact_last"
```

Comment:
Please use `@id/` instead of `@+id` when referring to an Id

Reply:
Yes, I'll change it.